docs/install/docker.md

```diff
@@ -54,16 +54,27 @@ an PostgreSQL container and volume.
 
 3. Start Coder with `docker-compose up`:
 
-   In order to use cloud-based templates (e.g. Kubernetes, AWS), you must set `CODER_ACCESS_URL` to the external URL that users and workspaces will use to connect to Coder.
+   In order to use cloud-based templates (e.g. Kubernetes, AWS), you must have an external URL that users and workspaces will use to connect to Coder.
+   
+   For proof-of-concept deployments, you can use [Coder's tunnel](../admin/configure.md#tunnel):
 
-   ```console
+   ```sh
+   cd coder
+
+   CODER_TUNNEL=true
+   docker-compose up
```

Comment:
I think these need to be on the same line, at least when I copy/pasted on macOS.
```suggestion
   CODER_TUNNEL=true docker-compose up
```